docs/source/reference/indexing.rst

```diff
@@ -49,6 +49,7 @@ Modifying and computations
    Index.is_interval
    Index.is_numeric
    Index.is_object
+   Index.drop_duplicates
```

Comment:
Sorry, can you add `drop_duplicates` at `MultiIndex` too?

Reply:
`MultiIndex.drop_duplicates` has been deprecated in pandas 0.26 doc.

Reply:
Nice!